config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
```

Comment:
I think it would be so very lovely to remove the comments from this initializer since it is long and realistically we will revisit the documentation to update the `simple_form` configuration anyway (not to mention that these comments will become outdated as we upgrade our version of `simple_form` over time)